rclpy/rclpy/time.py

```diff
@@ -40,6 +40,10 @@ def __init__(self, *, seconds=0, nanoseconds=0, clock_type=ClockType.SYSTEM_TIME
     def nanoseconds(self):
         return _rclpy.rclpy_time_point_get_nanoseconds(self._time_handle)
 
+    def seconds_nanoseconds(self):
+        nanoseconds = self.nanoseconds()
```

Comment:
TypeError: 'int' object is not callable
Should be `self.nanoseconds` instead of `self.nanoseconds()` ?

Reply:
Absolutely, fixed in 4e4255bd9b07aca611af743c4ca76b52490b2c56.